docs/en/interfaces/formats.md

```diff
@@ -325,6 +325,8 @@ Unlike the JSON format, there is no substitution of invalid UTF-8 sequences. Any
 
 For parsing, any order is supported for the values of different columns. It is acceptable for some values to be omitted – they are treated as equal to their default values. In this case, zeros and blank rows are used as default values. Complex values that could be specified in the table are not supported as defaults. Whitespace between elements is ignored. If a comma is placed after the objects, it is ignored. Objects don't necessarily have to be separated by new lines.
 
+Turn on calculation of columns with default expressions for omitted values in inserting data is possible by option `insert_sample_with_metadata=1`.
```

Comment:
It's better to modify the previous paragraph, because it says that omitted values are treated as default values and it's not clear why this additional setting is needed.

Reply:
Moved info about option inside previous paragraph for more clarity.